enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -693,7 +756,7 @@ func UnpackContainerImage(ctx context.Context, l *FileCacheLoader, imageRef, ima
 	// ChunkedFile then add it to cache.
 	// TODO(bduffany): single-flight this.
 	start := time.Now()
-	cow, err := copy_on_write.ConvertFileToCOW(ctx, l.env, imageExt4Path, chunkSize, outDir, key.InstanceName)
+	cow, err := copy_on_write.ConvertFileToCOW(ctx, l.env, imageExt4Path, chunkSize, outDir, "" /*=instanceName*/)
```

Comment:
why the removal of instance name here?

Reply:
Our current "non-COW" OCI -> ext4 conversion does not have instance name in the cache key so I think it was unintentional when I introduced instance name into the cache key originally.

Do you think we should include the instance name?

Reply:
Maybe add a comment that htis is nil to match OCI->ext4 conversion?

Reply:
Done